.github/actions/version-matrix/src/shared/constants.ts

```diff
@@ -1 +1,7 @@
 export const supportedPythonVersions = ['3.9', '3.10', '3.11', '3.12', '3.13'];
+
+export const supportedNodeVersions = ['18', '20', '22'];
+
+export const shouldUseLastFive = process.env.SHOULD_USE_LAST_FIVE === 'true';
```

Comment:
Is it a constant if it depends on an environment variable? :thinking: It probably doesn't matter much anyway...

Reply:
It's mostly so this is defined once and re-checked across files, so in the future if we add in more logic (or the boolean is a number, or the sun is a deadly lazor), we just update one spot